src/components/Comments/WriteComment.js

```diff
@@ -68,7 +68,7 @@ const WriteComment = ({
     if (isKeyboardOpen) {
       commentInputRef.current.focus();
     }
-  }, [isKeyboardOpen, commentInputRef.current]);
+  }, [isKeyboardOpen, commentInputRef?.current]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of optional chaining in the dependency array of `useEffect` is unnecessary and could lead to unexpected behavior. The `useEffect` hook compares dependencies (values in the dependency array) between renders using strict equality (`===`). If `commentInputRef.current` is `undefined`, it will be strictly equal to `undefined` on every render, so there's no need for optional chaining here.

```diff
-  }, [isKeyboardOpen, commentInputRef?.current]);
+  }, [isKeyboardOpen, commentInputRef.current]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->